server/data/statsClient.js

```diff
@@ -56,7 +56,8 @@ function createInitialCategoryOutcomesQuery(startDate, endDate, prisonId) {
 function createTprsTotalsQuery(catType, startDate, endDate, prisonId) {
   return {
     text: `select count(*)
-                    filter ( where form_response -> 'openConditions' -> 'tprs' ->> 'tprsSelected' = 'Yes' ) as tprs_selected
+                    filter ( where form_response -> 'openConditions' -> 'tprs' ->> 'tprsSelected' = 'Yes' and
+                                   form_response -> 'recat' -> 'decision' ->> 'category' = 'D' ) as tprs_selected
```

Comment:
based on this :

https://github.com/ministryofjustice/offender-categorisation/pull/684/files#diff-68b3150e62bf486684621bca0a4d11c4735d92965c0ea858814083821f008102R200

Does this need to check if D or J or T ?